benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -273,7 +290,7 @@ msgstr "Choose your transit provider"
 
 #: benefits/core/views.py:70
 msgid "core.pages.index.continue"
-msgstr "Let’s do it!"
+msgstr "Get started"
```

Comment:
Since this was changed, the specs that use `Let’s do it!` need to be updated to `Get started`

Reply:
Thanks! Gets me every time

I fixed this in 474b5b7